mlir-sycl/lib/Dialect/IR/SYCLOpsDialect.cpp

```diff
@@ -15,11 +15,15 @@
 #include "mlir/Dialect/SYCL/IR/SYCLOpsTypes.h"
 #include "mlir/IR/DialectImplementation.h"
 
+#include "llvm/ADT/TypeSwitch.h"
```

Comment:
Was going to use it, but not anymore. Removing it.

Reply:
**EDIT:** It's needed by `SYCLOpsTypes.cpp.inc` (included here). We need to include it here (instead of where the types are defined) because it's needed by `addTypes`.